### PR TITLE
Negar/fixes

### PR DESCRIPTION
**Pull request by negar-binary**
remove rise fall table and pricing table

### CODE REVIEW

File: src/templates/haml/resources/index.html.haml [hunk 1]
```diff
@@ -30,22 +30,3 @@
...
-        %img#charting_icon.responsive.charting-page-icon{:src => url_for('images/pages/resources/pricing-table.svg')}
-    .grd-grid-12
-      %h4
-        %a.pjaxload{:href => url_for('/resources/pricing_table')}= l('Pricing Table')
-      %p= l('View contract prices across a range of barriers and durations.')
-.grd-grid-12.grd-grid-mobile-12.grd-parent.grd-clear
-  .grd-grid-3.grd-grid-mobile-6.grd-parent.ja-hide
-    .grd-grid-8.grd-with-padding
-      %a.pjaxload{:href => url_for('/resources/rise_fall_table'),:id => 'contract-guide'}
-        %img#charting_icon.responsive.charting-page-icon{:src => url_for('images/pages/resources/rise-fall-table.svg')}
```
Comment (raunakkathuria) [inline src/templates/haml/resources/index.html.haml]: do we want to keep svg files?

**Comment by raunakkathuria** [comment]
I don't see any css changes, are we not removing it as of now?

File: src/templates/haml/resources/index.html.haml (hunk 1 above)
Comment (negar-binary) [inline src/templates/haml/resources/index.html.haml]: @raunakkathuria I asked Ashkan, we'll keep it for now

**Comment by negar-binary** [comment]
@raunakkathuria added, thanks

File: src/templates/haml/resources/index.html.haml (hunk 1 above)
Comment (negar-binary) [inline src/templates/haml/resources/index.html.haml]: @raunakkathuria deleted :)

**Pull request merged**